core/src/main/java/io/grpc/util/GracefulSwitchLoadBalancer.java

```diff
@@ -60,13 +60,13 @@ public String toString() {
 
   private final Helper helper;
 
-  // While the new policy is not fully switched on, the pendingLb is handling new updates from name
+  // While the new factory is not fully switched on, the pendingLb is handling new updates from name
```

Comment:
I feel like "policy" sounds better.

Reply:
I agree. I changed just be cause it was literally factory. Now I define policy by factory.